sysutils/pid_path_others.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build !darwin,!linux
+
+package sysutils
+
+// NotImplementedError is returned on platforms where GetExecPathFromPID is not
+// implemented.
+type NotImplementedError struct{}
```

Comment:
Should this be defined in another file, so that other code can check for it specifically without needing `!darwin,!linux`?  Either that or it probably shouldn't be exported.